tools/wrangling/xlsx2tsv.py

```diff
@@ -0,0 +1,33 @@
+import argparse
+
+import pandas as pd
+
+
+def convert_xlsx_to_tsv(input_file, sheet_name, output):
+    try:
+        # Read the specified sheet and convert them to tsv
+        df = pd.read_excel(input_file, sheet_name=sheet_name)
+        output_filename = f"{sheet_name}.tsv"
```

Comment:
`output_filename` is not really used. Maybe just print `Extracted sheet '{sheet_name}' from {input_file}`